### ISSUE TITLE
Exceptions on Python 3.x

### ISSUE THREAD

**Issue by mcgid** [mcgid/morenines#1]
A user had problems when running on Python 3.5:

```
~/Pictures/Wallpaper❯ mn init
SUCCESS: Initialized empty morenines repository in /Users/user/Pictures/Wallpaper/.morenines
~/Pictures/Wallpaper❯ mn create
SUCCESS: Wrote index file /Users/user/Pictures/Wallpaper/.morenines/index
~/Pictures/Wallpaper❯ mn status
Traceback (most recent call last):
  File "/usr/local/bin/mn", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.5/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.5/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.5/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/usr/local/lib/python3.5/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/morenines/application.py", line 113, in status
    new_files, missing_files, ignored_files = get_new_and_missing(repo, show_ignored)
  File "/usr/local/lib/python3.5/site-packages/morenines/util.py", line 61, in get_new_and_missing
    missing_files = [path for path in repo.index.files.iterkeys() if path not in current_files]
AttributeError: 'dict' object has no attribute 'iterkeys'
```

**Comment by mcgid** [comment]
This is because we're using `dict.iterkeys()` and `dict.iteritems()`, which don't exist in Python 3.x.

The iterator versions of these methods are more efficient because they don't construct an entirely new dict, but that isn't a huge concern here. (Until profiling proves otherwise, that is. But that's a long way off.) 

While we could continue to use the `iter` versions by fencing them in a `try/except`, it's much more straightforward to just use the non-`iter` versions.

**Comment by mcgid** [comment]
Fixed in 1.0.1 by using the non-`iter` versions of the methods.
